spec/complex_spec.rb

```diff
@@ -13,12 +13,50 @@
       end
       context 'with a Ruby Integer as input' do
         it 'returns an instance of SymEngine::Integer class' do
-          a = Complex(2, 0);
+          a = Complex(2, 0)
           a = SymEngine::sympify(a)
           expect(a).to be_an_instance_of SymEngine::Integer
           expect(a.to_s).to eq('2')
         end
       end
+      context 'Obtaining the integer real part' do
+        it 'returns an instance of SymEngine::Integer class' do
+          a = Complex(2, 7)
+          a = SymEngine::S(a)
+          a = a.real
```

Comment:
It's bad practice to re-assign variables in side a single unit test. `r = a.real` would be better.